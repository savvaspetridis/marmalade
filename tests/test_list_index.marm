/*
Tests reference list item by index
*/

/* 
measure swag = $() [55.q, 56.q];
note out = swag&0;

(print()) [(swag&0)];



int_list i = [42, 12, 13];
int i2 = i&2;



(print()) [i&2]; 

*/

str_list str = ["hello", "Hello", "World"];
string s = str&1;

(print()) [s];


/*
int_list swagOut = (print()) [out];
*/
