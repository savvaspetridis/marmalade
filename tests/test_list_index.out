jMusic NOTE: [Pitch = 56][RhythmValue = 0.5][Dynamic = 85][Duration = 0.45][Pan = 0.5]